src/pipecat/services/inworld/tts.py

```diff
@@ -689,15 +715,26 @@ async def _process_messages(self):
                 await self.push_error(error_msg=str(msg["error"]))
                 continue
 
-            # Skip messages for unavailable contexts
+            # Check if this message belongs to an available context.
+            # If the context isn't available but matches our current context ID,
+            # recreate it (handles race conditions during interruption recovery).
             if ctx_id and not self.audio_context_available(ctx_id):
-                continue
+                if self._context_id == ctx_id:
+                    logger.debug(
+                        f"{self}: Recreating audio context for current context: {self._context_id}"
+                    )
+                    await self.create_audio_context(self._context_id)
+                else:
+                    # This is a message from an old/closed context - skip it
+                    logger.debug(f"{self}: Skipping message from unavailable context: {ctx_id}")
+                    continue
 
             # Process audio chunk
             audio_chunk = result.get("audioChunk", {})
             audio_b64 = audio_chunk.get("audioContent")
 
             if audio_b64:
+                logger.debug(f"{self}: Processing audio chunk for context {ctx_id}")
                 await self.stop_ttfb_metrics()
                 self.start_word_timestamps()
```

Comment:
This must be awaited. I think you're behind `main`. This was caught by my colleague already and fixed on `main`.

```suggestion
                await self.start_word_timestamps()
```